src/common/settings_objects.cpp

```diff
@@ -220,110 +219,73 @@ namespace PowerToysSettings {
   // Resource helper.
   std::wstring Settings::get_resource(UINT resource_id) {
     if (resource_id != 0) {
-      wchar_t buffer[512];
-      if (LoadString(m_instance, resource_id, buffer, ARRAYSIZE(buffer)) > 0) {
-        return std::wstring(buffer);
+      std::aligned_storage_t<sizeof(void*)> res_ptr;
```

Comment:
why not have `wchar_t*` here?

Reply:
@bzoz to be more explicit - I'll change it to `wchar_t*`.